src/contexts/SearchProvider.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+
+export enum TypeFilter {
+  ALL,
+  DONE,
+  PLANNED,
```

Comment:
```suggestion
  ALL = "all",
  DONE = "completed", 
  PLANNED = "active",
```
after this update, you can specify the value in the options